invader_payment/models/payment_transaction.py

```diff
@@ -157,7 +157,7 @@ def _set_transaction_pending(self):
         self.write(
             {
                 "state": "pending",
-                "date": datetime.now().strftime(
+                "date_validate": datetime.now().strftime(
```

Comment:
@acsonefho IMO we must also replace `datetime.now().strftime...`. with `fields.Datetime.now()`